### PR TITLE
whitepapers: Update Governor whitepaper

### PR DESCRIPTION
**Pull request by tbjump**
* expand section explaining the reasons for defense-in-depth
* add clarity around limit vs. delay
* add FAQ section

### CODE REVIEW

File: whitepapers/0007_governor.md [hunk 1]
```diff
@@ -4,37 +4,88 @@
...
+If token bridge transfers are unlimited and instantaneous, a bug in a single connected chain could cause the entire token bridge to be drained.
 
-## Overview
+## Goals
 
-Each individual Guardian within the Guardian network should employ a set of strategies to verify the validity of a VAA. The Governor is designed to check VAAs that transfer tokens by enforcing limits on the notional value that can be transferred from a given chain over a specific period of time.
+If a Guardian decides to enable this feature:
+* Delay Wormhole messages from registered token bridges for 24h, if their notional value is excessively large.
+* This gives the Guardian the opportunity to delete pending messages, if they were created through a software bug and not accurately represent the state of the origin chain.
+* Protect against sybil attacks.
```
Comment (bruce-riley) [inline whitepapers/0007_governor.md:31]: Can you expand this sentence so people don't need to google "sybil attach"?

File: whitepapers/0007_governor.md [hunk 2]
```diff
@@ -4,37 +4,88 @@
...
-  - If a small transaction exceeds the `dailyLimit`it will be delayed until it either
-    - fits inside the `dailyLimit` and will be counted toward the `dailyLimit`
-    - has been delayed for 24-hours and will have no affect on the `dailyLimit`.
+* Synchronize state between Guardians. Each Guardian may have a slightly different view of the network, including Governor configuration, ordering and completeness of token bridge transfers, etc. The Governor is a local feature that makes decisions based on each Guardian's individual view of the network.
+* Prevent quality-of-service degradation attacks where one bad actor causes a majority of transfers to be delayed.
+
+## High-Level Design
+
+### Delay Decision Logic
+*Configuration*: For each chain, a _single-transaction-threshold_ and a *24h-threshold* denominated in a base currency (e.g. USD) is specified [here](https://github.com/wormhole-foundation/wormhole/blob/main/node/pkg/governor/mainnet_chains.go).
```
Comment (bruce-riley) [inline whitepapers/0007_governor.md:41]: Can you explicitly state that the governor only monitors chains and tokens that are in the configuration?

File: whitepapers/0007_governor.md [hunk 3]
```diff
@@ -4,37 +4,88 @@
...
+- **Is it possible to have my transfer delayed by Governor?**
+    - The above answer explains that transactions can be delayed during high utilization times.  However, the maximum possible wait time is 24hrs, though most transactions have no delay.
```
Comment (bruce-riley) [inline whitepapers/0007_governor.md:82]: Is it worth mentioning that the guardian operator can manually extend the 24 hour timer if they need more time to investigate?

**Review by bruce-riley — APPROVED** [review]

**Review by hendrikhofstadt — COMMENTED** [review]
I'm not sure these types of FAQs live in a design doc. They are probably a better fit for documentation as they just restate the more technical explanation in layman's terms

**Review by bruce-riley — APPROVED** [review]

**Review by evan-gray — CHANGES_REQUESTED** [review]
Similar to @hendrikhofstadt 's comment, I just enabled the wiki feature, perhaps that will be a good home for a plain english description of features.

**Comment by tbjump** [comment]
Addressed reviewer feedback: Removed the FAQ section

**Review by bruce-riley — APPROVED** [review]

**Review by evan-gray — APPROVED** [review]
